envFiles/.env.devcontainer

```
@@ -168,7 +174,19 @@ API_OTEL_SERVICE_NAME=talawa-api
-API_EMAIL_PROVIDER=ses
+# Use 'mailpit' for local development email testing (default)
+# Use 'ses' for AWS SES (requires AWS credentials)
+# Use 'smtp' for external SMTP server
+API_EMAIL_PROVIDER=mailpit
+
+# Mailpit SMTP settings (used when API_EMAIL_PROVIDER=mailpit)
+# Mailpit is a local email testing tool - no credentials needed
+SMTP_HOST=mailpit
+SMTP_PORT=1025
+SMTP_FROM_EMAIL=test@talawa.local
+SMTP_FROM_NAME=Talawa Dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMTP_FROM_NAME key should go before the SMTP_HOST key

(UnorderedKey)